### PR TITLE
Update orca pytorch quickstart

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: docs/docs/Orca/orca-pytorch-quickstart.md [hunk 1]
```diff
@@ -27,26 +20,32 @@ conda install pytorch torchvision -c pytorch # command for macOS
...
-sc = init_orca_context(
+if args.cluster_mode == "local":
+    init_orca_context(cores=1, memory="2g")   # run in local mode
+elif args.cluster_mode == "yarn":
+    init_orca_context(cluster_mode="k8s", num_nodes=2, cores=4) # run on K8s cluster
+elif args.cluster_mode == "yarn":
+    init_orca_context(
     cluster_mode="yarn-client", cores=4, num_nodes=2, memory="2g",
     driver_memory="10g", driver_cores=1,
     conf={"spark.rpc.message.maxSize": "1024",
```
Comment (jason-dai) [inline docs/docs/Orca/orca-pytorch-quickstart.md:32]: can we make these config default @qiuxin2012 @hkvision ?

File: docs/docs/Orca/orca-pytorch-quickstart.md [hunk 2]
```diff
@@ -1,23 +1,16 @@
...
 
 ```bash
-conda create -n zoo python=3.7 # zoo is conda enviroment name, you can set another name you like.
+conda create -n zoo python=3.7 # zoo is conda environment name, you can use any name you like.
 conda activate zoo
-pip install analytics_zoo-${VERSION}-${TIMESTAMP}-py2.py3-none-${OS}_x86_64.whl 
+pip install analytics_zoo-${VERSION} # install either version 0.9 or latest nightly build
 pip install jep==3.9.0
 conda install pytorch torchvision cpuonly -c pytorch # command for linux
 conda install pytorch torchvision -c pytorch # command for macOS
```
Comment (jason-dai) [inline docs/docs/Orca/orca-pytorch-quickstart.md:16]: which version have we tested our PyTorch code on?

File: docs/docs/Orca/orca-pytorch-quickstart.md [hunk 3]
```diff
@@ -27,26 +20,32 @@ conda install pytorch torchvision -c pytorch # command for macOS
...
         "spark.task.maxFailures": "1",
-        "spark.driver.extraJavaOptions": "-Dbigdl.failure.retryTimes=1"})
+        "spark.driver.extraJavaOptions": "-Dbigdl.failure.retryTimes=1"})   # run on Hadoop YARN cluster
 ```
-**Note:** You should `export HADOOP_CONF_DIR=/path/to/hadoop/conf/dir`. 
 
-View [Orca Context](./context) for more details.
+This is the only place where you need to specify local or distributed mode. View [Orca Context](./context) for more details.
+
+**Note:** You should `export HADOOP_CONF_DIR=/path/to/hadoop/conf/dir`.
```
Comment (jason-dai) [inline docs/docs/Orca/orca-pytorch-quickstart.md:39]: Why is this needed? For YARN cluster?

File: docs/docs/Orca/orca-pytorch-quickstart.md [hunk 4]
```diff
@@ -69,12 +68,13 @@ class LeNet(nn.Module):
 model = LeNet()
 model.train()
 criterion = nn.NLLLoss()
-adam = Adam(1e-4)
+adam = torch.optim.Adam(model.parameters(), args.lr)
```
Comment (hkvision) [inline docs/docs/Orca/orca-pytorch-quickstart.md:71]: Better not use args in quick start, just give a default value.

File: docs/docs/Orca/orca-pytorch-quickstart.md (hunk 1 above)
Comment (hkvision) [reply docs/docs/Orca/orca-pytorch-quickstart.md:32]: I think `spark.task.maxFailures` would affect the Spark tasks?
Already opened an issue #3151 pending for discussion.

File: docs/docs/Orca/orca-pytorch-quickstart.md (hunk 3 above)
Comment (cyita) [reply docs/docs/Orca/orca-pytorch-quickstart.md:39]: Yes, YARN cluster need this env.

File: docs/docs/Orca/orca-pytorch-quickstart.md (hunk 2 above)
Comment (cyita) [reply docs/docs/Orca/orca-pytorch-quickstart.md:16]: I tested on torch 1.7.0 and torchvision 0.8.1.

File: docs/docs/Orca/orca-pytorch-quickstart.md (hunk 4 above)
Comment (cyita) [reply docs/docs/Orca/orca-pytorch-quickstart.md]: Fixed

File: docs/docs/Orca/orca-pytorch-quickstart.md (hunk 3 above)
Comment (jason-dai) [reply docs/docs/Orca/orca-pytorch-quickstart.md]: Do we need this for TF and Keras quickstart?

Comment (cyita) [reply docs/docs/Orca/orca-pytorch-quickstart.md]: Yes, HADOOP_CONF_DIR is needed on YARN cluster

**Comment by jason-dai** [comment]
LGTM

**Comment by cyita** [comment]
Jenkins passed http://10.239.47.210:18888/job/ZOO-PR-Validation/4867/
